cmake/modules/OmrTargetSupport.cmake

```diff
@@ -0,0 +1,86 @@
+###############################################################################
+# Copyright (c) 2020, 2020 IBM Corp. and others
+#
+# This program and the accompanying materials are made available under
+# the terms of the Eclipse Public License 2.0 which accompanies this
+# distribution and is available at http://eclipse.org/legal/epl-2.0
+# or the Apache License, Version 2.0 which accompanies this distribution
+# and is available at https://www.apache.org/licenses/LICENSE-2.0.
+#
+# This Source Code may also be made available under the following Secondary
+# Licenses when the conditions for such availability set forth in the
+# Eclipse Public License, v. 2.0 are satisfied: GNU General Public License,
+# version 2 with the GNU Classpath Exception [1] and GNU General Public
+# License, version 2 with the OpenJDK Assembly Exception [2].
+#
+# [1] https://www.gnu.org/software/classpath/license.html
+# [2] http://openjdk.java.net/legal/assembly-exception.html
+#
+# SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+#############################################################################
+
+if(OMR_TARGET_SUPPORT)
+	return()
+endif()
+set(OMR_TARGET_SUPPORT 1)
+
+include(OmrAssert)
+include(OmrUtility)
+
+# omr_add_library(name <sources> ...)
+#   STATIC | SHARED | OBJECT  - same meanings as for standard add_library
+# At present, a thin wrapper around add_library, but it ensures that exports
+# and split debug info are handled
+function(omr_add_library name)
```

Comment:
File should be indented with tabs (sorry)

Reply:
what I really need to do is to setup my vscode to default to tabs on cmake files